index.js

```diff
@@ -28,16 +28,29 @@ module.exports = function (RED) {
 
         node.on('input', function (msg) {
             var parameterName = config.parameter || msg.parameter;
-            var realValue = msg.payload;
-            pump.write(parameterName, realValue, function (err, data) {
-                if (err) {
-                    msg.payload = err;
-                    return node.send(msg);
-                }
-                msg.payload = data;
-                node.send(msg);
-            });
+            var parameterValue = msg.payload;
+            var rawParameterNumber = msg.raw_parameter;
```

Comment:
rawParameter instead of rawParameterNumber?

Reply:
Makes sense.
I renamed it.

Reply:
@coolchip Did you already check the little patch?